src/data/device_automation.ts

```diff
@@ -57,7 +66,7 @@ export const fetchDeviceTriggerCapabilities = (
     trigger,
   });
 
-const whitelist = ["above", "below", "for"];
+const whitelist = ["above", "below", "code", "for"];
```

Comment:
This might get a bit out of hand..
Maybe it would be better to pass the automation's extra_fields list to deviceAutomationsEqual instead?

Reply:
Yeah, agree

Reply:
The problem is just that HaDeviceAutomationPicker doesn't know about the extra fields right now.